src/engine/order.ts

```diff
@@ -133,9 +135,11 @@ export class OrderEngine {
 
     const rawQuote = side === 'buy' ? await this.quoteEngine.getBuyPrice() : await this.quoteEngine.getSellPrice()
     const quote = this.exchangeProvider.priceToPrecision(exchange, symbol, rawQuote)
+    const adjustedQuote = side === 'buy' ? quote - this.orderSlippageAdjustmentTolerance : quote + this.orderSlippageAdjustmentTolerance
```

Comment:
Do you really want a real number here? I would prefer a percentage instead.

Reply:
I thought a static number would be better. Since it's checking against the price, which is usually a fiat currency, so setting it to 0.5 would be a 50 cent slippage tolerance. Whereas if it was using a percentage then you'd never be able to get that level of granular control.

I can put in both and have 1 take precedence over the other.

Reply:
With percent, you'd have to set quite a few leading zeros:
BTC = 6450
`6450 * 0.0001 = .645`

Reply:
i would disagree that its usually fiat currency, trading in a pair like BTC/ETH or something like this is (from my experience) the more common way, but if it was your intention we could keep it for the moment.
To add both it makes the use case more complex and i would then just prefer the current version.